it/apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
     <filter root="/apps/forms-core-components-it"/>
+    <filter root="/apps/core"/>
```

Comment:
Please correct the filter, all it related collateral should be inside `/apps/forms-core-components-it`